WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/logic/interval/IntervalPersister.java

```diff
@@ -79,11 +82,26 @@ private void recreateDatabase(final File file) {
 	}
 
 	private void deleteDatabaseFile() {
-		file.delete();
-		File axuiliaryDatabaseFile = new File(file + ".p");
-		axuiliaryDatabaseFile.delete();
-		axuiliaryDatabaseFile = new File(file + ".t");
-		axuiliaryDatabaseFile.delete();
+		emptyFile(file, null);
+		emptyFile(file, ".p");
+		emptyFile(file, ".t");
+	}
+
+	private void emptyFile(File file, String extension) {
```

Comment:
Because I didn't sleep well last night and I'm a bit out of focus, sorry. I'll fix this immediately, thanks for pointing this out!